README_zh-CN.md

```diff
@@ -126,79 +106,123 @@ mim install -e .
 
 我们为新用户提供了一系列基础教程：
 
-- [使用现有模型推理](https://mmclassification.readthedocs.io/zh_CN/1.x/user_guides/inference.html)
-- [准备数据集](https://mmclassification.readthedocs.io/zh_CN/1.x/user_guides/dataset_prepare.html)
-- [训练与测试](https://mmclassification.readthedocs.io/zh_CN/1.x/user_guides/train_test.html)
 - [学习配置文件](https://mmclassification.readthedocs.io/zh_CN/1.x/user_guides/config.html)
-- [如何微调模型](https://mmclassification.readthedocs.io/zh_CN/1.x/user_guides/finetune.html)
-- [分析工具](https://mmclassification.readthedocs.io/zh_CN/1.x/user_guides/analysis.html)
-- [可视化工具](https://mmclassification.readthedocs.io/zh_CN/1.x/user_guides/visualization.html)
-- [其他工具](https://mmclassification.readthedocs.io/zh_CN/1.x/user_guides/useful_tools.html)
+- [准备数据集](https://mmclassification.readthedocs.io/zh_CN/1.x/user_guides/dataset_prepare.html)
+- [使用现有模型推理](https://mmclassification.readthedocs.io/zh_CN/1.x/user_guides/inference.html)
+- [训练](https://mmclassification.readthedocs.io/zh_CN/pretrain/user_guides/train.html)
+- [测试](https://mmclassification.readthedocs.io/zh_CN/pretrain/user_guides/test.html)
+- [下游任务](https://mmclassification.readthedocs.io/zh_CN/pretrain/user_guides/downstream.html)
```

Comment:
why some of the links use 1.x, some of links use pretrain

Reply:
I just add the link and plan to update all readthedocs links after the readthedoc being renamed to mmpretrain